### ISSUE TITLE
Query defaults to LUCENE parsing when lucene index is available

### ISSUE THREAD

**Issue by lasagnaman** [orientechnologies/orientdb#6959]
## OrientDB Version, operating system, or hardware.
- [x] v2.2.4

## Operating System
- [x] Linux
- [ ] MacOSX
- [ ] Windows
- [ ] Other Unix
- [ ] Other, name? 

## Expected behavior and actual behavior
Except where specified, we are using com.tinkerpop.blueprints.impls.orient.OrientBaseGraph and associated APIs.

We have a vertex class called Foo, with a single property "value". We have two indexes called Foo.value, and Foo.value_fulltext_lucene. The first is an SBTREE UNIQUE index, and the latter is a LUCENE FULLTEXT index; both have the keyproperty "value". We make a call to `OrientBaseGraph.getVertices(final String iKey, Object iValue)`, with parameters `iKey = "Foo.value"` `and iValue = "blahblah / foobar"`. (This vertex exists in the DB and can be verified through OrientDB Studio.)

**Expected behavior:**
Returns 1 vertex

**Actual Behavior:**
```
Exception in thread "main" com.orientechnologies.orient.core.index.OIndexEngineException: Error parsing lucene query
<…full traceback snipped…>
Caused by: com.orientechnologies.orient.core.sql.parser.ParseException: Cannot parse 'blahblah / foobar': Lexical error at line 1, column 36.  Encountered: <EOF> after : "/ foobar"
```

**Comments:**
It seems like the query value is being treated as a Lucene query, and the parser breaks down because the Lucene regex doesn't have a closing / character. If I properly escape the / character, then nothing "breaks", but it's still being parsed as a Lucene query and returns hundreds of records. But I don't want to be using a Lucene query here! Any ideas?

* If I query a Vertex class/index that does NOT have a Lucene index (e.g. has only the SBTREE UNIQUE index Bar.value) then the query works as expected. In this case, / is not treated as a special Lucene character, it returns a single vertex, etc.
* The same (broken) behavior happens if I use OrientBaseGraph.getVertices(final String label, final String[] iKey, Object[] iValue).

**Comment by lasagnaman** [comment]
@luigidellaquila says this will be fixed in 2.2.14.

**Comment by luigidellaquila** [comment]
I can confirm, it's already fixed in 2.2.x branch

Thanks

Luigi
